ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB.hs

```diff
@@ -85,54 +66,19 @@ prop_roundTrips ls = property $ (binarySize * (length ls), ls) === (fromIntegral
                     sl = fromBlock $ Binary.decode bs1
                 in sl : go bsRest
 
--- Write some blocks and then try to get them back.
-prop_VolatileRoundTrips :: HasCallStack => [MyBlockId] -> Property
-prop_VolatileRoundTrips ls = monadicIO $ do
-    let ls' = (\sl -> (sl, Binary.encode $ toBlock sl)) <$> ls
-    let expected :: [Either String MyBlockId] = (Right <$> ls)
-    run $ apiEquivalenceVolDB (expectVolDBResult (@?= expected)) (\hasFS err ->
-        withTestDB hasFS err ["demo"] myParser 5 $ \db -> do
-            forM_ ls' $ \(slot, enc) -> putBlock db slot (BL.lazyByteString enc)
-            bss <- forM ls $ \slot -> getBlock db slot
-            return $ (fmap fromBlock) <$> decode <$> catMaybes bss
-        )
-
--- Check if the db works normaly after reopening.
-prop_VolatileReOpenWrite :: HasCallStack => [MyBlockId] -> [MyBlockId] -> Property
-prop_VolatileReOpenWrite ls1 ls2 = monadicIO $ do
-    let ls1' = (\sl -> (sl, Binary.encode $ toBlock sl)) <$> ls1
-    let ls2' = (\sl -> (sl, Binary.encode $ toBlock sl)) <$> ls2
-    let expected :: [Either String MyBlockId] = Right <$> (ls1 <> ls2)
-    run $ apiEquivalenceVolDB (expectVolDBResult (@?= expected)) (\hasFS err -> do
-                db <- openDB hasFS err ["reopen"] myParser 5 toSlot
-                forM_ ls1' $ \(slot, enc) -> putBlock db slot (BL.lazyByteString enc)
-                closeDB db
-                reOpenDB db
-                forM_ ls2' $ \(slot, enc) -> putBlock db slot (BL.lazyByteString enc)
-                ret <- forM (ls1 <> ls2) $ \slot -> getBlock db slot
-                closeDB db
-                return $ (fmap fromBlock) <$> decode <$> catMaybes ret
+prop_VolatileInvalidArg :: HasCallStack => Property
```

Comment:
This can't be replaces from q-s-m since it checks whether the db fails to open with invalid arguments.